src/main/java/org/isf/patient/rest/PatientController.java

```diff
@@ -78,9 +84,17 @@ public PatientController(PatientBrowserManager patientManager, AdmissionBrowserM
 		this.admissionManager = admissionManager;
 		this.patientMapper = patientMapper;
 	}
+	
+	public PatientController(PatientBrowserManager patientManager, AdmissionBrowserManager admissionManager, PatientMapper patientMapper, PatientConsensusBrowserManager patientConsensusManager) {
+		this.patientManager = patientManager;
+		this.admissionManager = admissionManager;
+		this.patientMapper = patientMapper;
+		this.patientConsensusManager = patientConsensusManager;
+	}
```

Comment:
it seems that to have two constructors confuses the ApplicationContext:
```
Error starting ApplicationContext. To display the conditions report re-run your application with 'debug' enabled.
[02/may./2023 18:51:44] [:] ERROR - Application run failed
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'patientController' defined in file [D:\IDE\workspace.2022-03\openhospital-api\target\classes\org\isf\patient\rest\PatientController.class]: Instantiation of bean failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.isf.patient.rest.PatientController]: No default constructor found; nested exception is java.lang.NoSuchMethodException: org.isf.patient.rest.PatientController.<init>()
```

Do we really need to add a second constructor rather than modifying the former?
By commenting the new constructor application runs well (test not, of course) and all endpoints tests are ok (using Postman). Also using current UI (develop) is ok! 👏